testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -2280,7 +2277,7 @@ fn size_overflow_unconfirmed_stream_microblocks_integration_test() {
     let mut ctr = 0;
     while ctr < flat_txs.len() {
         submit_tx(&http_origin, &flat_txs[ctr]);
-        if !wait_for_microblocks(&microblocks_processed, 240) {
+        if !wait_for_microblocks(&microblocks_processed, 60) {
```

Comment:
Due to the time it takes for gigantic microblocks to process, 60 seconds isn't enough to run it locally.  It really takes about 4 minutes.